src/main/java/org/javarosa/form/api/FormEntryCaption.java

```diff
@@ -265,6 +266,16 @@ public String getRepeatText(String typeKey) {
         return form.fillTemplateString(caption, index.getReference(), vars);
     }
 
+    private String getCaptionText(String textIdOrText) {
```

Comment:
I think this also needs to be used for `entryHeader` and `chooseCaption` in `getRepetitionText`?

Reply:
Ahh well spotted, https://github.com/dimagi/commcare-core/pull/1428/commits/58b8829fed95e7ae660017b47e6059c204569c39